client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.jsx

```diff
@@ -0,0 +1,273 @@
+						ref={ statusContentRef }
+						onMouseEnter={ handleShowTooltip }
+						onMouseLeave={ handleHideTooltip }
+					>
+						{ updatedContent }
+					</span>
+					<Tooltip
+						id={ tooltipId }
+						context={ statusContentRef.current }
+						isVisible={ showTooltip }
+						position="bottom"
+						className="sites-overview__tooltip"
+					>
+						{ tooltip }
+					</Tooltip>
+				</>
+			) : (
+				updatedContent
+			) }
+		</>
+	);
+}
+
+const statusFormatter = ( rows, type ) => {
+	const { link, row, siteError, tooltip, tooltipId, eventId } = getRowMetaData( rows, type );
+	const { value, status } = row;
+	let content;
+	switch ( status ) {
+		case 'failed': {
+			content = (
+				<Badge className="sites-overview__badge" type="error">
+					{ value }
+				</Badge>
+			);
+			break;
+		}
+		case 'warning': {
+			content = (
+				<Badge className="sites-overview__badge" type="warning">
+					{ value }
+				</Badge>
+			);
+			break;
+		}
+		case 'success': {
+			content = <Gridicon icon="checkmark" size={ 18 } className="sites-overview__grey-icon" />;
+			break;
+		}
+		case 'active': {
+			content = <Gridicon icon="minus-small" size={ 18 } className="sites-overview__icon-active" />;
+			break;
+		}
+		case 'progress': {
+			content = (
+				<svg
+					className="sites-overview__vertical-align-middle"
+					width="16"
+					height="16"
+					viewBox="0 0 16 16"
+					fill="none"
+					xmlns="http://www.w3.org/2000/svg"
+				>
+					<circle cx="8" cy="8" r="7.25" stroke="black" strokeWidth="1.5" />
+					<rect x="7" y="4" width="1.5" height="5" fill="#1E1E1E" />
+					<rect
+						x="10.8901"
+						y="10.77"
+						width="1.5"
+						height="4"
+						transform="rotate(135 10.8901 10.77)"
+						fill="#1E1E1E"
+					/>
+				</svg>
+			);
+			break;
+		}
+		case 'inactive': {
+			content = (
+				<span className="sites-overview__status-add-new">
+					<Gridicon icon="plus-small" size={ 16 } />
+					<span>{ translate( 'Add' ) }</span>
+				</span>
+			);
+			break;
+		}
+	}
+	return (
+		<StatusContent
+			content={ content }
+			tooltip={ tooltip }
+			tooltipId={ tooltipId }
+			link={ link }
+			siteError={ siteError }
+			type={ type }
+			eventId={ eventId }
+		/>
+	);
+};
+
+const siteFormatter = ( rows ) => {
+	const { row } = getRowMetaData( rows, 'site' );
+	const site = row.value;
+	const value = site.url;
+
+	return <span className="sites-overview__row-text">{ value }</span>;
+};
+
+export const formatSites = ( data ) => {
+	const sites = data?.items;
+	if ( Array.isArray( sites ) && sites.length > 0 ) {
+		return sites.map( ( site ) => {
+			const pluginUpdates = site.awaiting_plugin_updates;
+			let scanValue = '';
+			if ( site.latest_scan_status === 'failed' ) {
+				scanValue = translate( 'Failed' );
+				if ( site.latest_scan_threats_found.length > 0 ) {
+					scanValue = translate( '%(threats) Threats', {
+						args: {
+							threats: site.latest_scan_threats_found.length,
+						},
+					} );
+				}
+			}
```

Comment:
> I think we display threats even if the latest scan run failed since previous scans could have found threats in the past.

If scan threats aren't fixed yet, I think the following scan should also include them, I think?